modules/cluster/main.tf

```diff
@@ -117,7 +117,7 @@ resource "null_resource" "kubeconfig" {
     module.eks
   ]
   provisioner "local-exec" {
-    command     = "aws eks update-kubeconfig --name ${var.cluster_name}"
+    command     = "aws eks update-kubeconfig --region us-east-1 --name ${var.cluster_name}"
```

Comment:
I dont think this is the root cause of the issue. What happens if the eks cluster is in ca-central-1 or any region other than us-east-1.
I think there are 2 ways to fix this:
* Explicitely specify a region in the provider block in the main.tf:
```
provider "aws" {
  region  = "us-east-1"
}
```
* Specify region as an environment variable:
```
AWS_REGION=us-east-1 terraform apply
```

Reply:
And if you are using AWS_PROFILE, make sure there is a matching `config` file with the region for that profile set

Reply:
I also addressed this in #102.